### PR TITLE
BaSP-TG-32: Create Table Component

### PR DESCRIPTION
**Pull request by PilarFernandezC**
- Create generic table
- Add styles with CSS for table

### CODE REVIEW

File: src/Components/Shared/Table/table.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.table {
+    background-color: #3738671a;
+    box-shadow: inset 0px 2px 0px #E0E0E0;
+    text-transform: capitalize;
+    border-radius: 5px;
+    border-collapse: collapse;
+    border: solid 2px #3738671a;;
```
Comment (Maxi-SR) [inline src/Components/Shared/Table/table.module.css:7]: double semicolon here

Comment (PilarFernandezC) [reply src/Components/Shared/Table/table.module.css]: done

**Review by Maxi-SR — APPROVED** [review]

File: src/Components/Shared/Table/index.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import styles from './table.module.css';
+import Row from '../Row';
```
Comment (Juanoauil) [inline src/Components/Shared/Table/index.jsx:2]: is this already merge?

Comment (PilarFernandezC) [reply src/Components/Shared/Table/index.jsx:2]: No, I have to wait for that to be ready before I merge this PR

**Review by DamianPalavecino — APPROVED** [review]
Good work Pilar!

**Review by martumoser — APPROVED** [review]

**Pull request merged**